transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CctTransportBackend.java

```diff
@@ -130,6 +133,20 @@ private static TelephonyManager getTelephonyManager(Context context) {
     return (TelephonyManager) context.getSystemService(Context.TELEPHONY_SERVICE);
   }
 
+  private static int getVersionCode(Context context) {
+    try {
+      int packageVersionCode =
+          context
+              .getPackageManager()
+              .getPackageInfo(context.getPackageName(), /* flags= */ 0)
+              .versionCode;
+      return packageVersionCode;
+    } catch (PackageManager.NameNotFoundException e) {
+      Logging.e(LOG_TAG, "Couldn't find version code", e);
```

Comment:
Do you want to qualify this error a bit better? "Unable to find version code for package - <package name>". This would enable the developer fix quickly rather than being too abstract.

Reply:
Sounds good! Fixed.